test/src/unit-sparse-global-order-reader.cc

```diff
@@ -1423,16 +1423,20 @@ TEST_CASE_METHOD(
     run<Asserter>(instance);
   };
 
-  SECTION("Example") {
-    doit.operator()<tiledb::test::AsserterCatch>(200, 8, 2);
-  }
-
-  SECTION("Condition") {
-    int value = 110;
-    tdb_unique_ptr<tiledb::sm::ASTNode> qc(new tiledb::sm::ASTNodeVal(
-        "a1", &value, sizeof(int), tiledb::sm::QueryConditionOp::GE));
-    doit.operator()<tiledb::test::AsserterCatch>(200, 8, 2, {}, std::move(qc));
-  }
+  // CORE-328: Tests that fail on nightly windows-latest - Sanitizer: OFF |
+  // Assertions: ON | Debug To re-enable when fixed.
+  //
+  // SECTION("Example") {
+  //   doit.operator()<tiledb::test::AsserterCatch>(200, 8, 2);
```

Comment:
We can use `SKIP("CORE-328 ...")` to skip single sections in test cases for known issues, it gives some output from the test suite for the location and reason

```
/home/shaun/Code/WIP/tiledb/main/test/src/unit-s3.cc:346: SKIPPED:
explicitly with message:
  Test skipping a single SECTION
```

Reply:
The reason I didn't go for that solution is that I found Catch2 handling of skipped sections messy having read the [documentation](https://catch2-temp.readthedocs.io/en/latest/skipping-passing-failing.html#interaction-with-sections-and-generators). 

I did try it here since this change was marked as required and indeed just adding SKIP returns a non-zero code that makes CI fail (Error: Process completed with exit code 4. [src](https://github.com/TileDB-Inc/TileDB/actions/runs/16564593089/job/46841506486#step:14:42278)). I found some relavant open issues here: https://github.com/catchorg/Catch2/issues/2873 and here: https://github.com/catchorg/Catch2/issues/1146

I am trying a couple of workarounds and I'll update here.

Reply:
I didn't have any luck with those workarounds. I feel I am spending too much time today on working around just disabling a couple of tests, so I'd appreciate any suggestions/ideas on how to circumvent this odd behavior of SKIP section in our system. Otherwise I'd suggest to fallback on the initial implementation to get the purpose of this PR fulfilled.

cc : @rroelke